__tests__/fixtures/tailwind-output-important.css

```diff
@@ -14411,7 +14398,7 @@ video {
   }
 
   50% {
-    transform: translateY(0);
+    transform: none;
```

Comment:
What's the benefit of this one aside from just file size? This is the only part that makes me a bit nervous since transitioning from translateY(-25%) to none feels like it might not work in some browsers. Reminds me of how Safari can't transition from a hex color to "transparent" or something (can't remember exactly).

Reply:
> What's the benefit of this one aside from just file size?

Just file size I guess (and `translateY(0)` looks a bit odd to me). Keep in mind that the other changes in this PR, eg. the removal of `from { transform: rotate(0deg); }` in the spin animation, actually do the same. Instead of using `rotate(0deg)`, it uses the default transform state, probably also `none`

> This is the only part that makes me a bit nervous since transitioning from translateY(-25%) to none feels like it might not work in some browsers.

BS uses `none` too, no browser issues have been reported yet. Apart from https://github.com/twbs/bootstrap/issues/30493, which was an issue with using `50%` as end state (actually not related to this at all).

> Reminds me of how Safari can't transition from a hex color to "transparent" or something (can't remember exactly).

Probably this issue: Some browsers interpret `transparent` as `rgba(255, 255, 255, 0)` which causes animations from `white` to darken when they fade out.
This isn't the case with transforms though.

Reply:
Awesome good to know, this is great then! Thanks so much man ❤️